atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -959,6 +959,17 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
             }
         }
 
+        if (AtlasDbConstants.hiddenTables.contains(tableRef)) {
```

Comment:
this should be covered by ``checkGetPreconditions`` in most pubic get APIs, looks like ``getRowsColumRange`` misses this. We should add it there.